pkg/skaffold/docker/image_test.go

```diff
@@ -312,6 +312,16 @@ func TestGetBuildArgs(t *testing.T) {
 			},
 			want: []string{"--no-cache"},
 		},
+		{
+			description: "secret",
+			artifact: &latest.DockerArtifact{
+				Secret: &latest.DockerSecret{
+					ID:     "mysecret",
+					Source: "foo.bar",
+				},
+			},
+			want: []string{"--secret", "id=mysecret,src=foo.bar"},
```

Comment:
Add another test sInce `src`/`Source` is  optional.

Reply:
done